apps/web/app/(app)/gardens/page.tsx

```diff
@@ -88,20 +60,22 @@ export default async function Gardens() {
       </header>
       <section className="my-2 flex w-full max-w-2xl flex-col items-center justify-center gap-8">
         <div className="grid max-w-7xl grid-cols-[repeat(auto-fit,minmax(310px,1fr))] gap-6 md:grid-cols-[repeat(auto-fit,minmax(320px,1fr))]">
-          {gardens ? (
-            gardens.tokenGardens.map((garden, id) => (
-              <GardenCard garden={garden} key={`${garden.id}-${id}`} />
+          {!gardens ? (
+            <LoadingSpinner />
+          ) : tokenGardens?.length ? (
+            tokenGardens.map((garden, id) => (
+              <div key={`${garden.id}-${id}`}>
+                <GardenCard garden={garden} />
+              </div>
```

Comment:
why not just `gardens && tokensGardens?.length ?` check and else been the Spinner?
That way its confusing and looks like do the same, right?

Reply:
Problem is if no gardens its infinite spinner

Reply:
if not gardens you have else with Spinner

Reply:
This is personal opinion I think haha

Reply:
Something like this would be clearer? 
![image](https://github.com/1Hive/gardens-v2/assets/13890618/d60dab93-7e59-4651-82d8-1427df7c1b91)

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/0ca7beb7ca26727226a969b0b007c87c07c80a57